content/blog/2022/10/2022-10-10-plugin-health-scoring-system-report.adoc

```diff
@@ -0,0 +1,110 @@
+* Jenkins Online Meetup Phase 1: link:https://docs.google.com/presentation/d/1t2vuNn1NFpDusnw0m4vdFw6WBQMeU6kccv_K1v2L6R0/edit#slide=id.g13dcaed2105_0_0[Slides]
+* Jenkins Online Meetup Phase 1: Demo and Presentation
+
+video::loLSNdCv6K4[youtube,width=800,height=420,start=1089]
+
+---
+
+== Phase 2
+
+In this phase, the probe engine for the probes was designed. And processes like reading the update center and executing the probe engine are now running on a schedule without the need to restart the application. There are additions of useful probes running in the probe engine, which are getting listed on the UI to highlight the current state of the project.
+
+=== Deliverables
+
+* Design the Probe engine for the probes (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/19[PR])
+* Schedule the reading of the update center, and probe engine execution (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/20[PR#1] and link:https://github.com/jenkins-infra/plugin-health-scoring/pull/30[PR#2])
+* List the available probes of the application (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/27[PR])
+* Add a probe (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/33[PR])
+* Visit link:https://plugin-health.jenkins.io[plugin-health.jenkins.io] to view a list of active probes. Thanks to the Jenkins Infrastructure team
+
+=== Resources
+
+* link:https://github.com/jenkins-infra/plugin-health-scoring/milestone/2?closed=1[View Milestone 2]
+* Jenkins Online Meetup Phase 2: link:https://docs.google.com/presentation/d/1HOHRVFOfH07TnBfbGh3xAqakA3NfmKni_7FYyCx-llw/edit#slide=id.p[Slides]
+* Jenkins Online Meetup Phase 2: Demo and Presentation
+
+video::fM2SMbppRxw[youtube,width=800,height=420,start=342]
+
+* A BIG shoutout to my mentor link:https://www.jenkins.io/blog/authors/jleon/[Jake Leon], for devoting his time to coaching me so well for both of these presentations.
+
+---
+
+== Next Steps
+
+- Add more probes to the project.
+- Generate the plugin health scores based on the data extracted by the probes.
+- Deploy the health scores via a JSON file, similar to how Jenkins Update Center does it.
+- Render the detailed report of the health score of each plugin by fetching the JSON data generated above.
+- [Stretch Goal] Display Plugin health score on Plugin Manager.
+
+---
+
+== Useful Links
+
+- link:https://plugin-health.jenkins.io[plugin-health.jenkins.io]
+- link:https://github.com/jenkins-infra/plugin-health-scoring[GitHub repository]
+- link:https://docs.google.com/document/d/1Dxyli1LPlHdFxLoE9zFtr_3bTjnwQDMZGCxcGS79Z_I/edit[Architecture Diagram]
+- link:https://docs.google.com/document/d/1HTbcWh5C1KrCgEzgqeVEPyfr1H5fH5eTj8KpbWrWsSY/edit#heading=h.efprktbggbop[GSoC Proposal Document]
+- Use the link:https://gitter.im/jenkinsci/GSoC-Plugin_Health_Score[Gitter channel] or link:https://community.jenkins.io[community.jenkins.io] in case you have any question(s) or feedback.
```

Comment:
I've attached the link to the Gitter channel (https://gitter.im/jenkinsci/GSoC-Plugin_Health_Score) here, so that interested contributors can use it to ask their questions there. As you can see in the channel, we are planning to create a new channel for this project whose context is not just limited to GSoC.

I can create a new PR in the future that updates this link to point to the new channel (if needed) or we'll post the new channel's link in this current channel, just like a redirect for the visitors). If you have any concerns about this, please let me know.